src/core_modules/capture-core/components/WidgetNote/NoteSection/NoteSection.tsx

```diff
@@ -114,19 +118,18 @@ const NoteSectionPlain = ({
                     </span>
                 </div>
                 <div className={classes.body}>
-                    <Parser>{value}</Parser>
+                    <Parser>{note || value}</Parser>
                 </div>
             </div>
         </div>
     );
 
-
     return (
         <div className={classes.wrapper}>
             <div className={classes.notesWrapper}>
                 {notes
                     .sort((a, b) => moment(a.storedAt).valueOf() - moment(b.storedAt).valueOf())
-                    .map(note => <NoteItem key={`note-item-${note.note}-`} {...note} />)
+                    .map(note => <NoteItem key={`note-item-${note.value || note.note}-`} {...note} />)
```

Comment:
First use `note.note` as the default and `note.value` as the fallback. Like this `{note.note || note.value}`